int pedestrian_id
time[] times
geometry_msgs/Point[] trajectory
